eng/scripts/automation/GenerateAndBuildLib.ps1

```diff
@@ -755,14 +755,14 @@ function GeneratePackage()
     } else {
         # Build
         Write-Host "Start to build sdk: $projectFolder"
-        dotnet build $projectFolder /p:RunApiCompat=$false
+        dotnet build $projectFolder
```

Comment:
@ArthurMa1978 @live1206 I think this may impact your sdk generation steps. Didn't you want to disable this during that process for some reason that I cannot remember now.

Reply:
/cc @m-nash as well